requirements.txt

```diff
@@ -1,7 +1,8 @@
 # >=1.24.2 from kubernetes-asyncio (from API requirements), <1.25 from kfp
 urllib3>=1.24.2, <1.25
 GitPython>=2.1.0
-aiohttp>=3.5.0
+# required by google-auth
+aiohttp<4.0.0dev,>=3.6.2
```

Comment:
where this came from ?

Reply:
Real breakage (this is what @quaark hit last week).
There is a newer aiohttp version released (4.0.0a1 which is an alpha release) which is incompatible with the requirements of google-auth (like the comment says). So I put the requirement of the google-auth package as is, and now the alpha release is not picked up